lib/features/modeling/cmd/UpdateFlowNodeRefsHandler.js

```diff
@@ -101,8 +98,6 @@ UpdateFlowNodeRefsHandler.prototype.computeUpdates = function(flowNodeShapes, la
     });
 
     children.forEach(addFlowNodeShape);
-
-    handledNodes[root.id] = root;
```

Comment:
Can we remove this without any replacement? It will iterate over the same children again for two lanes within the same participant.

Reply:
Actually it should not add the same children as the addFlowNodeShape does an indexOf check and also at the beginning of the method we do an early return also if the allFlowNodeShapes contain root. Does that make sense to you? Or maybe I misunderstood your point?